microsite/pages/en/plugins.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const fs = require('fs')
+const yaml = require('js-yaml');
+const React = require('react');
+const Components = require(`${process.cwd()}/core/Components.js`);
+const { Block: { Container }, BulletLine } = Components;
+
+const pluginsDirectory = require('path').join(process.cwd(), 'data/plugins');
+const pluginMetadata = fs.readdirSync(pluginsDirectory)
+  .map((file) => yaml.safeLoad(fs.readFileSync(`./data/plugins/${file}`, 'utf8')));
+const truncate = (text) => (text.length > 170 ? text.substr(0, 170) + '...' : text);
+
+const addPluginDocsLink = 'https://github.com/spotify/backstage/blob/master/microsite/README.md#adding-a-plugin-to-the-plugins-page';
```

Comment:
I would prefer that this documentation lives in the main docs on the microsite. My suggestion is to add another page next to https://backstage.io/docs/plugins/publishing, something like https://backstage.io/docs/plugins/add-to-marketplace

Reply:
I did consider this. The reason I didn't was that from the point of view of the microsite repo it's README should contain all the necessary details for making changes to it and I didn't want to duplicate information. We could create a page which summarises the process and links to this README but then that seems like an unnecessary step.

Reply:
We are trying to stay away from keeping documentation inside various folders in the repo. People simply don't discover them as easily and it is harder to provide a "flow" in our docs.

Reply:
Fair enough.